middleware/auth.js

```diff
@@ -68,10 +68,10 @@ module.exports = function (params) {
   });
 
   if (config.routes) {
-    router.get('/login', (req, res) => {
+    router.get(config.loginPath, (req, res) => {
```

Comment:
I think it's missing the tests for this file changes. You are now testing the config options are being set, but you're not testing that they are being used in the router.